IdentityCore/src/cache/token/MSIDAccountCacheItem.m

```diff
@@ -179,6 +192,14 @@ - (NSDictionary *)jsonDictionary
     dictionary[MSID_REALM_CACHE_KEY] = _realm;
     dictionary[MSID_CLIENT_INFO_CACHE_KEY] = _clientInfo.rawClientInfo;
     dictionary[MSID_ALTERNATIVE_ACCOUNT_ID_KEY] = _alternativeAccountId;
+
+    // Last Modification info (currently used on macOS only)
+    if (_lastModificationTime)
+    {
+        dictionary[MSID_LAST_MOD_TIME_CACHE_KEY] = [NSString stringWithFormat:@"%0.3f", [_lastModificationTime timeIntervalSince1970]];
```

Comment:
use msidDateToTimestamp instead for consistency?

Reply:
moved this to NSDate+MSIDExtensions.m